### PR TITLE
feat(payment): PAYPAL-876 added preloader for APM when polling mechanism is running

### PR DESCRIPTION
**Pull request by andriiVitvitskyi1990**
## What?
Added preloader for APM when polling mechanism is running

## Why?
According to task PAYPAL-876

## Testing / Proof
Tested on dev

@bigcommerce/checkout @bigcommerce/payments @davidchin

### CODE REVIEW

**Review by capsula4 — APPROVED** [review]
👍

File: src/common/loading-indicator/loading-indicator.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { LoadingIndicatorStyles } from './embedded-checkout-styles';
+import { LoadingIndicatorStyles } from '../../embedded-checkout';
```
Comment (davidchin) [inline src/common/loading-indicator/loading-indicator.ts:1]: Can you move the `LoadingIndicatorStyles` to the `common/loading-indicator` folder as well?

File: src/payment/strategies/paypal-commerce/paypal-commerce-payment-strategy.ts [hunk 2]
```diff
@@ -23,9 +24,14 @@ const ORDER_STATUS_APPROVED = 'APPROVED';
 const ORDER_STATUS_CREATED = 'CREATED';
 const POLLING_INTERVAL = 3000;
 const POLLING_MAX_TIME = 600000;
+const LOADING_INDICATOR_CONTAINER = 'checkout-app';
```
Comment (davidchin) [inline src/payment/strategies/paypal-commerce/paypal-commerce-payment-strategy.ts:27]: The container ID should not be hard-coded. Can you find a way to pass it in?

**Review by davidchin — CHANGES_REQUESTED** [review]

File: src/common/loading-indicator/loading-indicator.ts (hunk 1 above)
Comment (andriiVitvitskyi1990) [reply src/common/loading-indicator/loading-indicator.ts]: moved

File: src/payment/strategies/paypal-commerce/paypal-commerce-payment-strategy.ts (hunk 2 above)
Comment (andriiVitvitskyi1990) [reply src/payment/strategies/paypal-commerce/paypal-commerce-payment-strategy.ts]: fixed

**Review by davidchin — APPROVED** [review]
Thank you 👍

**Pull request merged**